x-pack/filebeat/module/rabbitmq/log/ingest/pipeline.yml

```diff
@@ -0,0 +1,27 @@
+---
+description: Pipeline for parsing RabbitMQ logs
```

Comment:
Nice, you converted it to YAML 👍

Reply:
Yeah 😄 I started because of https://github.com/elastic/beats/pull/12032#discussion_r282393958 but I also realized it was the only config file in JSON. Now the only JSON file is the `-expected.json`